experimental/dds/attributable-map/src/test/mocha/map.spec.ts

```diff
@@ -771,85 +835,171 @@ describe("Map", () => {
 				});
 			});
 		});
-	});
 
-	describe("Garbage Collection", () => {
-		class GCSharedMapProvider implements IGCTestProvider {
-			private subMapCount = 0;
-			private _expectedRoutes: string[] = [];
-			private readonly map1: SharedMap;
-			private readonly map2: SharedMap;
-			private readonly containerRuntimeFactory: MockContainerRuntimeFactory;
+		describe("Attributor", () => {
+			beforeEach(() => {
+				const options: IMapOptions = { attribution: { track: true } };
+				containerRuntimeFactory = new MockContainerRuntimeFactory();
+				// Connect the first map with attribution enabled.
+				map1 = createConnectedMap("map1", containerRuntimeFactory, options);
+				// Create a second map with attribution enabled.
+				map2 = createConnectedMap("map2", containerRuntimeFactory, options);
+			});
 
-			public constructor() {
-				this.containerRuntimeFactory = new MockContainerRuntimeFactory();
-				this.map1 = createConnectedMap("map1", this.containerRuntimeFactory);
-				this.map2 = createConnectedMap("map2", this.containerRuntimeFactory);
-			}
+			it("Can retrieve proper attribution information with some normal operations", () => {
+				map1.set("key1", 1);
+				map1.set("key2", 2);
+				map2.set("key1", 3);
+				map1.delete("key2");
 
-			/**
-			 * {@inheritDoc @fluid-internal/test-dds-utils#IGCTestProvider.sharedObject}
-			 */
-			public get sharedObject(): SharedMap {
-				// Return the remote SharedMap because we want to verify its summary data.
-				return this.map2;
-			}
+				containerRuntimeFactory.processSomeMessages(1);
 
-			/**
-			 * {@inheritDoc @fluid-internal/test-dds-utils#IGCTestProvider.expectedOutboundRoutes}
-			 */
-			public get expectedOutboundRoutes(): string[] {
-				return this._expectedRoutes;
-			}
+				let attribution1 = map1.getAttribution("key1");
+				let attribution2 = map1.getAttribution("key2");
+				let attribution3 = map2.getAttribution("key1");
+				let attribution4 = map2.getAttribution("key2");
 
-			/**
-			 * {@inheritDoc @fluid-internal/test-dds-utils#IGCTestProvider.addOutboundRoutes}
-			 */
-			public async addOutboundRoutes(): Promise<void> {
-				const newSubMapId = `subMap-${++this.subMapCount}`;
-				const subMap = createLocalMap(newSubMapId);
-				this.map1.set(newSubMapId, subMap.handle);
-				this._expectedRoutes.push(subMap.handle.absolutePath);
-				this.containerRuntimeFactory.processAllMessages();
-			}
+				assert.equal(
```

Comment:
here and below, i'd find inlining the `attribution1` variables and using `assert.deepEqual` easier to read.

For example here it would be
```
assert.deepEqual(map1.getAttribution("key1"), { type: "op", seq: 1 });
```